js/state/siteUtil.js

```diff
@@ -371,10 +371,10 @@ module.exports.filterSitesRelativeTo = function (sites, relSite) {
 }
 
 /**
- * Clears out all sites that have no tags.
+ * Clears history (all sites that have no tags).
  * @param sites The application state's Immutable sites list.
  */
-module.exports.clearSitesWithoutTags = function (sites) {
+module.exports.clearHistory = function (sites) {
```

Comment:
if this clears history for all sites we should use the method here https://github.com/brave/browser-laptop/blob/master/app/sessionStore.js#L258